ui/src/message_popup/linear_session/linear_session.jsx

```diff
@@ -14,10 +14,22 @@ export default React.createClass({
   propTypes: {
     questions: React.PropTypes.array.isRequired,
     questionEl: React.PropTypes.func.isRequired,
+    getNextQuestion: React.PropTypes.func,
```

Comment:
This is a bit of a nitpick, but using `onQuestionNeeded` might be a event-y sounding name to signify that this is callback function.